app/templates/layouts/_base.html

```diff
@@ -133,4 +133,26 @@
       document.write("<script async nonce='{{ csp_nonce() }}' src='http://HOST:5075/browser-sync/browser-sync-client.js?v=2.24.4'><\/script>".replace("HOST", location.hostname));
     //]]></script>
     {% endif %}
+
+    <script nonce="{{ csp_nonce() }}" >
+        (function() {
+          var printed = 0;
```

Comment:
I think it would be clearer if this was a boolean instead of an integer.

Reply:
changed to boolean